internal/graph/dotbuilder.go

```diff
@@ -0,0 +1,49 @@
+package graph
+
+import (
+	"fmt"
+	"io"
+)
+
+const attr = `fontname="Helvetica,Arial,sans-serif"
```

Comment:
`dotLabelAttr`?

Reply:
Actually, we only need the node attributes since we're not writing any labels for the edges, so I'll just inline it.